airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_audits.json

```diff
@@ -144,10 +144,20 @@
             "type": ["null", "integer"]
           },
           "value": {
-            "type": ["null", "string", "array"],
-            "items": {
-              "type": ["null", "string"]
-            }
+            "oneOf": [
```

Comment:
pointing out that as per https://github.com/airbytehq/airbyte/issues/15021, `oneOf`s don't work well with normalization so we should move towards emitting values of a single type.

I won't block this PR on this because it's fixing an OC issue and the normalization problem was already present with the previous type definition.

Reply:
yes I have similar PR https://github.com/airbytehq/airbyte/pull/15049 where I did similar thing

I have re-implemented here 
from `["null", "string", "array"]` to `oneOf {"null", "string", "array" }`
to keep the same baheviour on normalization level 
but `oneOf` works better with python Transformer 

yes I know that we have plans to eliminate all multi-types